### PR TITLE
Employees can view all or single gif posted or also delete gif

### PR DESCRIPTION
**Pull request by goodnessaig1**

### CODE REVIEW

File: server/controller/gifController.js [hunk 1]
```diff
@@ -53,6 +53,81 @@ class GifController {
...
+        error: err.message
+      });
+      }
+  }
+
+
+  static async getSingleGif(req, res) {
+    try {
+        const { gifId } = req.params;
+        const gif = await pool.query(`SELECT * FROM gifs WHERE gif_id = ${gifId}`);
```
Comment (courage173) [inline server/controller/gifController.js]: goodness we talked about this, parametized query please

File: server/queries/gifQuery.js [hunk 2]
```diff
@@ -1,10 +1,15 @@
 const createNewGif = 'INSERT INTO gifs (title, image_url, public_id, created_at, user_id) VALUES($1,$2,$3,$4, $5) RETURNING *';
-
+const selectGif = `SELECT * FROM gifs WHERE gif_id = $1`
```
Comment (courage173) [inline server/queries/gifQuery.js:2]: why did you not use this to select a single gif

**Review by courage173 — CHANGES_REQUESTED** [review]

File: server/controller/gifController.js (hunk 1 above)
Comment (goodnessaig1) [reply server/controller/gifController.js]: Sir it was a mistake o. I already created the query in my queries. But i did not move it there.

File: server/queries/gifQuery.js (hunk 2 above)
Comment (goodnessaig1) [reply server/queries/gifQuery.js:2]: Ok sir. I have used it

**Pull request merged**